test/Directory.Build.props

```diff
@@ -5,6 +5,7 @@
     <DefaultTestTargetFramework>netcoreapp2.1</DefaultTestTargetFramework>
     <DefaultTestTargetFramework Condition="$(Configuration.EndsWith('-netfx'))">net462</DefaultTestTargetFramework>
     <DefaultTestTargetFramework Condition="'$(UseIntrinsics)' == 'true'">netcoreapp3.0</DefaultTestTargetFramework>
+    <DefineConstants Condition="'$(DefaultTestTargetFramework)' == 'netcoreapp3.0'">NETCOREAPP30</DefineConstants>
```

Comment:
This isn't necessary - these DefineConstants are defined implicitly. 

Instead of `NETCOREAPP30` => `NETCOREAPP3_0`